Hardware/HardwareTests/AirPrintReceipt/ReceiptRendererTest.swift

```diff
@@ -5,11 +5,14 @@ import Foundation
 import CryptoKit
 
 final class ReceiptRendererTest: XCTestCase {
+    let locale = Locale(identifier: "en_US_POSIX")
+    let timeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
nit: these could be private

```suggestion
    private let locale = Locale(identifier: "en_US_POSIX")
    private let timeZone = TimeZone(secondsFromGMT: 0)!
```